### PR TITLE
Merge release 4.0.0 into master

### PR DESCRIPTION
**Pull request by wetransferplatform**
Containing all the changes for our [**4.0.0 Release**](https://github.com/WeTransfer/Diagnostics/releases/tag/4.0.0).

### CODE REVIEW

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/6e8cc404-704e-4656-bfde-09050383baf6" target="_blank">Bitrise</a>
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>DiagnosticsTests: Executed 30 tests (0 failed, 0 retried, 0 skipped) in 0.447 seconds</td>
    </tr>
  </tbody>
</table>

## Code Coverage Report
| Name | Coverage ||
| --- | --- | --- |
Diagnostics | 75.23% | ⚠️

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 65e32d74a8777b1737b2c369515e8cf6cdfb2ae6
</p>

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+### 4.0.0
+- Use String instead of StaticString to be more inline with swift-log behaviour ([#118](https://github.com/WeTransfer/Diagnostics/pull/118)) via [@antranapp](https://github.com/antranapp)
+- Adds Raphael as a code owner ([#117](https://github.com/WeTransfer/Diagnostics/pull/117)) via [@kairadiagne](https://github.com/kairadiagne)
+- Catch obj-c exception, remove unneeded dispatch ([#116](https://github.com/WeTransfer/Diagnostics/pull/116)) via [@AvdLee](https://github.com/AvdLee)
+- Update example and readme ([#115](https://github.com/WeTransfer/Diagnostics/pull/115)) via [@mikakruschel](https://github.com/mikakruschel)
+- Merge release 3.0.0 into master ([#114](https://github.com/WeTransfer/Diagnostics/pull/114)) via [@wetransferplatform](https://github.com/wetransferplatform)
```
Comment (BasThomas) [inline Changelog.md:6]: Hm, why is this included here? 🧐

Comment (AvdLee) [reply Changelog.md:6]: Because it was merged after we released 3.0.0, it was part of the 3.0.0 release train

**Review by AvdLee — APPROVED** [review]

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 4.1.0](https://github.com/WeTransfer/Diagnostics/releases/tag/4.1.0) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>
